src/validator/Validator.js

```diff
@@ -601,6 +617,7 @@ class ValidationChecker {
       },
       custom: {
         key: 'validate.custom',
+        hasLabel: true,
```

Comment:
We had someone complain about Custom including the labels as well. If they are adding a custom error message, wouldn't they also be able to add the label if they want it there?

Reply:
I didn't take into account the logic of custom validations. If I understood correctly, now everything should work correctly.